src/main/resources/openapiv2.json

```diff
@@ -0,0 +1,875 @@
+                                        },
+                                        {
+                                            "broker": 3,
+                                            "leader": false,
+                                            "in_sync": false
+                                        }
+                                    ]
+                                }
+                            ]
+                        }
+                    },
+                    "404": {
+                        "description": "Topic or Partition not found"
+                    },
+                    "422": {
+                        "description": "Unprocessable Entity"
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "topicname",
+                    "in": "path",
+                    "description": "Name of the topic",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "partitionid",
+                    "in": "path",
+                    "description": "ID of the partition",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/positions": {
+            "post": {
+                "tags": [
+                    "Consumers"
+                ],
+                "summary": "Overrides the fetch offsets that the consumer will use for the next set of records to fetch",
+                "operationId": "seek",
+                "parameters": [
+                    {
+                        "name": "body",
+                        "in": "body",
+                        "description": "A list of offsets",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/Offsets"
+                        }
+                    }
+                ],
+                "responses": {
+                    "204": {
+                        "description": "No Content"
+                    },
+                    "404": {
+                        "description": "Consumer instance not found"
+                    }
+                }
+            },
+            "parameters": [
+                {
+                    "name": "groupid",
+                    "in": "path",
+                    "description": "The name of the consumer group",
+                    "required": true,
+                    "type": "string"
+                },
+                {
+                    "name": "name",
+                    "in": "path",
+                    "description": "The ID of the consumer instance",
+                    "required": true,
+                    "type": "string"
+                }
+            ]
+        },
+        "/consumers/{groupid}/instances/{name}/offsets": {
```

Comment:
In my testing, the endpoint appeared to be named `offset` not `offsets`. Please check.